mle/model.py

```diff
@@ -189,6 +189,36 @@ def __init__(self, api_key, model, temperature=0.7):
         self.model_type = MODEL_CLAUDE
         self.temperature = temperature
         self.client = self.anthropic(api_key=api_key)
+        self.func_call_history = []
+
+    @staticmethod
+    def _add_tool_result_into_chat_history(chat_history, func, result):
```

Comment:
Is the function compatible with both gpts and claude models?

Reply:
no, only for claude models